debian/changelog

```diff
@@ -1,3 +1,212 @@
+  * adding full colour icon for drive-multidisk
+
+  [ Ken VanDine ]
+  * When building with communitheme_compat, link the dark variant as well
+  * When building with communitheme_compat, link the light variant
+
+  [ chrisjbillington ]
+  * Ensure shell-theme link can be created when using DESTDIR
+  * Include shell themes in the shell Debian package, not the gtk package.
+
+  [ Feichtmeier ]
+  * Remove dowstream check/radio flattening
+  * Darken progress blue in dark themes
+  * Gtk3 upstream refresh containing...
+  * Medica optical changes
+  * Add a color for the finger print label
+  * Remove text and iconshadows for headerbar and btns
+  * Lighten headerbar border-color for the dark theme
+  * Vanilla color changes without touching common
+  * Button and color changes and comments
+  * Color and button changes
+  * Remove the headerbar gradient
+  * Color improvements
+  * Color and focus ring changes
+  * New upstream changes applied on _common.scss
+  * Reorganize colors and includes
+  * Reorganize colors and includes
+  * Green color improvements
+  * brighten up dark menus
+  * Upstream updates in _common for...
+  * Improve ubiquity dot grid
+  * Style ubiquity dotgrid with scss
+  * Typography update
+  * Bring back undershoots
+  * Update _tweaks.scss
+  * Add missing underscores
+  * Fix circular button padding
+  * Sligtly lighten up the headerbar border for the dark theme
+  * Update _apps.scss
+  * Special case switch focus
+  * Outline radii fixes and tweaks cleanup
+  * Common: change outline raddi for linked classes
+  * OSD button styling
+  * Fix app notification buttons
+  * Fix hb focused entries
+  * Adapt gtk2 menucolor to gtk3
+  * Darken dark theme switch border
+  * Increase blue darkening by 4%
+  * Remove the dedicated hb styling from tweaks
+
+  [ Merlijn Sebrechts ]
+  * overwrite softlinks so install-compat can handle multiple builds
+
+  [ Leandro Vital ]
+  * fix dock background opacity to dynamic
+  * Empty class declaration, as in dash to dock
+
+  [ Didier Roche ]
+  * Automate upstream GNOME Shell refresh
+  * Update link to point to refresh github action
+  * Factor out automatic PR updates from new upstream version
+  * Add GTK auto refresh
+  * Enhance package description.
+  * Select branch for autorebase
+  * Try updating an existing PR
+  * Dummy ommit to trigger first PR update automation
+  * New upstream snapshot for GNOME Shell
+
+  [ Marco Trevisan (Treviño) ]
+  * gnome-shell: Sync with 3.34.2 changes
+
+  [ github-actions[bot] ]
+  * New upstream snapshot for GTK
+
+  [ MadsRH ]
+  * Fix symlink badge for new folder icons (PR for 1742)
+  * Added cursor test
+  * Update README.md
+  * Mimetype icons for .PO translation files
+  * Mimetype icon for RSS
+  * Mimetype for application-x-python-bytecode to replace the Humanity fallback
+  * fixed missing "eyes and stroke covering the Python logo
+  * more contrast for context menus
+  * reverted popover
```

Comment:
IMHO many of these entries don't make sense to someone reading the Debian changelog. Imagine if you have `apt-listchanges` installed and you are shown this when you do an upgrade. :)

I think it'd be best if there was something more descriptive - what changed that people will see? If there was a `NEWS` file announcing this new version, what would be in it? - or at least please could you delete the entries that aren't understandable like this one and many of the others?

Thanks!

Reply:
Yeah, I understand what you mean. I'll try to rewrite some entries